packages/shrinkpdf.rb

```diff
@@ -2,25 +2,22 @@
 
 class Shrinkpdf < Package
   description 'A simple wrapper around Ghostscript to shrink PDFs (as in reduce filesize) under Linux.'
-  homepage 'http://alfredklomp.com/programming/shrinkpdf/'
-  version '2014'
+  homepage 'https://github.com/aklomp/shrinkpdf'
+  version '1.1'
   license 'BSD-3'
   compatibility 'x86_64 aarch64 armv7l'
-  source_url 'http://alfredklomp.com/programming/shrinkpdf/index.html'
-  source_sha256 'cdc74b28d8ad757fa2cc9d8782031fe2325a17d3f4fd6080c05ca62790e5c690'
-  binary_compression 'tar.xz'
+  source_url 'https://raw.githubusercontent.com/aklomp/shrinkpdf/v#{version}/shrinkpdf.sh'
+  source_sha256 'bd23c0917741a6a63694bcb99315c4afabdac0d24fef2be896c5be5c7b3aed3c'
 
   binary_sha256({
-    aarch64: '19c53e4be836957a64613eb60e329506cfafde2891c54155b99ba16de0855ba6',
-     armv7l: '19c53e4be836957a64613eb60e329506cfafde2891c54155b99ba16de0855ba6',
-     x86_64: '92e7bddf8ead1324927547c1591e7f6192c2b195a8137a3365597c08d802f950'
+    aarch64: '?',
+     armv7l: '?',
+     x86_64: '?'
   })
```

Comment:
This bash executable is small so we don't really need binaries.  Probably can remove the whole section.  That said, you can replace the binary section with `no_compile_needed`.  Typically, we put these "boolean properties" below any depends_on lines.